mapdata/WarcraftLegacies/AbilityData/Skin/1918986305.json

```diff
@@ -29,7 +29,7 @@
       "Level": 2,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Harvests lumber from trees."
+      "Value": "Harvest Gold from gold mines"
```

Comment:
Missed this ability! (Both this tooltip and the one below).

Reply:
Ditto comment